core/templates/dev/head/components/forms/AudioTranslationsEditorDirective.js

```diff
@@ -137,6 +139,8 @@ oppia.directive('audioTranslationsEditor', [
 
                   $scope.save = function() {
                     if ($scope.isAudioTranslationValid()) {
+                      $scope.saveButtonText = 'Saving...'
+                      $scope.saveInProgress = true;
```

Comment:
Does this need to be reset after successfully saving audio, too?

Reply:
I don't think so because in that case the modal just closes anyway